docs/examples/basic-price-oracle/go.mod

```diff
@@ -20,7 +20,10 @@ require (
 	dario.cat/mergo v1.0.1 // indirect
 	github.com/Azure/go-ansiterm v0.0.0-20230124172434-306776ec8161 // indirect
 	github.com/Microsoft/go-winio v0.6.1 // indirect
+	github.com/NebulousLabs/fastrand v0.0.0-20181203155948-6fb6489aac4e // indirect
```

Comment:
While we're at it: can we please replace the `github.com/lightninglabs/taproot-assets => ../../../` at the top with a fixed version so we don't have to update these two files with almost every PR?
I know it risks us forgetting to update it if we ever actually change anything that impacts the example. But the number of force pushes and CI cycles that can be avoided by forgetting to update these two files IMO is worth it.

Reply:
I've pinned to the latest `main` commit of taproot-assets for now because that was the easiest to build. Once the next release is out, I will pin to that. See commit message for more. Let me know if there's a better strategy.